docs/developer-docs/latest/developer-resources/cli/CLI.md

```diff
@@ -115,25 +115,35 @@ The exported file is automatically named using the format `export_YYYYMMDDHHMMSS
 | `--no-compress`    |     -    | Disables file compression.                                                                                |
 | `-k`, `--key`            | string  | Passes the encryption key as part of the `export` command. <br/> The `--key` option can't be combined with `--no-encrypt`. |                                |
 | `-f`, `--file`       | string  | Specifies the export filename. Do not include a file extension.                                           |
+| `--exclude` | string | Exclude data using comma-separated data types. `--exclude` types override `--only` types. The available types are: `content`, `files`, and `config`.|
+| `--only`| string | Include only these data. The available types are: `content`, `files`, and `config`.|
```

Comment:
In the latest version I've added a note to --only that it will always include schemas, since those are necessary.